<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tulostaPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tulosta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="showTextLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="languageGroupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>120</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Kielivalinta</string>
    </property>
    <widget class="QRadioButton" name="suomiRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Suomi</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="englishRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Englanti</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="tulostusStatusLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ei tulostettu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="varoitaPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Varoitus</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>showTextLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
